Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -710,6 +710,11 @@ open class LottieAnimationView: LottieAnimationViewBase {
     animationLayer?.getOriginalValue(for: keypath, atFrame: atFrame)
   }
 
+  /// Gets all child keypaths.
+  public func getHierarchyKeypaths() -> [String] {
+    animationLayer != nil ? animationLayer!.getHierarchyKeypaths() : []
```

Comment:
No need to check for nil / force unwrap:

```suggestion
    animationLayer?.getHierarchyKeypaths() ?? []
```